outline.md

```diff
@@ -0,0 +1,17 @@
+## Motivation
+
+Introduce the document by describing the motivations for producing a consolidated set of software citation principles. This section is basically an overview of the different use cases/scenarios where it would be desirable to have a way to cite software.
+
```

Comment:
What's the best way of presenting the overview of use cases in a way that makes it quick to get to the Software citation principles, and check-off against the principles in the justification? Is a tabular format appropriate, or something else?